syntax = "proto3";
package booking;

service Booker {
    rpc OrderPlace (Order) returns (Response);
}

message Order {
    int32 event_id = 1;
    string username = 2;
    int32 places = 3;
}

message Response {
    int32 event_id = 1;
    int32 order_places = 2;
    bool success = 3;
}